classes/db/Db.php

```diff
@@ -819,7 +944,7 @@ public static function checkEncoding($server, $user, $pwd)
      */
     public static function hasTableWithSamePrefix($server, $user, $pwd, $db, $prefix)
     {
-        return call_user_func_array(array(Db::getClass(), 'hasTableWithSamePrefix'), array($server, $user, $pwd, $db, $prefix));
+        return call_user_func_array([Db::getClass(), 'hasTableWithSamePrefix'], [$server, $user, $pwd, $db, $prefix]);
```

Comment:
@firstred can you use the old ``array(...)`` notation please ? same for others `[ ]` :)